etcdserver/raft_test.go

```diff
@@ -22,12 +22,12 @@ import (
 	"time"
 
 	"go.etcd.io/etcd/etcdserver/api/membership"
+	"go.etcd.io/etcd/etcdserver/api/v2store"
```

Comment:
do we really need to introduce a dependency on v2 stuff in this test?

Reply:
thanks, no need.